source/sass/buyers-guide/views/catalog.scss

```diff
@@ -477,6 +472,8 @@ body.catalog {
     the privacy ding when "show pni only" is on:
   */
   &.show-ding-only figure.product-box:not(.privacy-ding) {
+    // we have to use !important here because .product-box uses Bootstrap's .d-flex
```

Comment:
https://foundation-s-issue-6355-xnjp28.herokuapp.com/en/privacynotincluded/

This is for the filtered product list to work when the "*privacy not included" checkbox is on 
<img width="246" alt="image" src="https://user-images.githubusercontent.com/2896608/112393461-556cd900-8cb8-11eb-9a84-de33d6687a8a.png">